.github/workflows/bazel.yml

```diff
@@ -209,7 +209,7 @@ jobs:
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_TITLE: "Bazel Build & Test Job `bazel build //...; bazel test //...` ${{ matrix.bazel-config }}"
```

Comment:
While issue #14323 talks only about workflow names, would it make sense to do the same for job names in the Slack title? CC @voisey

Reply:
That was part of what I intended, yes. Sorry for the lack of clarity!

Reply:
Basically, I think everything should be unified as far as possible

Reply:
The thing with the job names is, that they are [not generically available inside the context](https://stackoverflow.com/questions/71593861/how-to-get-github-actions-matrix-job-name-within-job), but only the `job_id` is. :cry: